apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -674,126 +671,123 @@ namespace MWMechanics
     }
 }
 
-
 namespace
 {
-
-ESM::Weapon::AttackType chooseBestAttack(const ESM::Weapon* weapon, MWMechanics::Movement &movement)
-{
-    ESM::Weapon::AttackType attackType;
-
-    if (weapon == NULL)
+    ESM::Weapon::AttackType chooseBestAttack(const ESM::Weapon* weapon, MWMechanics::Movement &movement)
     {
-        //hand-to-hand deal equal damage for each type
-        float roll = Misc::Rng::rollClosedProbability();
-        if(roll <= 0.333f)  //side punch
-        {
-            movement.mPosition[0] = Misc::Rng::rollClosedProbability() ? 1.0f : -1.0f;
-            movement.mPosition[1] = 0;
-            attackType = ESM::Weapon::AT_Slash;
-        }
-        else if(roll <= 0.666f) //forward punch
+        ESM::Weapon::AttackType attackType;
+
+        if (weapon == NULL)
         {
-            movement.mPosition[1] = 1;
-            attackType = ESM::Weapon::AT_Thrust;
+            //hand-to-hand deal equal damage for each type
+            float roll = Misc::Rng::rollClosedProbability();
+            if(roll <= 0.333f)  //side punch
+            {
+                movement.mPosition[0] = Misc::Rng::rollClosedProbability() ? 1.0f : -1.0f;
```

Comment:
Indeed the behaviour was changed in this commit: 3f28634d1f617691c672e41a3ee950e6daec8c77